pkg/acpi/tables.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
+
+package acpi
+
+import "fmt"
+
+// GetTable uses all the Methods until one works.
+func GetTable() (string, []Table, error) {
+	for m, f := range Methods {
+		t, err := f()
+		if err == nil {
+			return m, t, nil
+		}
+	}
+	return "", nil, fmt.Errorf("Could not get tables")
```

Comment:
lower case error strings

Reply:
also, please be more specific. "could not get ACPI tables"

I can imagine this being embedded in some iSCSI boot error string, and seeing "iSCSI boot failed: could not get tables" which is really unspecific